extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -684,14 +684,14 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
-   * @param {!Object<string, Object<string, string|Array<string>>>} source1
-   * @param {!Object<string, Object<string, string|Array<string>>>} source2
+   * @param {!JsonObject|!./events.AnalyticsEvent} source1
```

Comment:
mergeObject requires an `Object`. Does `JsonObject` and `Object` equals in type check? 
@erwinmombay do we want to avoid using object in the future?